src/main/scala/com/databricks/spark/avro/package.scala

```diff
@@ -44,4 +49,68 @@ package object avro {
   implicit class AvroDataFrameReader(reader: DataFrameReader) {
     def avro: String => DataFrame = reader.format("com.databricks.spark.avro").load
   }
+
+  /**
+   * Adds a method, `toRowRDD`, to RDDs of GenericRecords.  This allows you to convert it to
+   * an RDD of Row (and, by implication, a DataFrame).
+   */
+  implicit class GenericRecordRDD(rdd: RDD[GenericRecord]) {
+    def toRowRDD(requiredColumns: Array[String]): RDD[Row] = rdd.mapPartitions { records =>
```

Comment:
As long as we are going to expose this, we should also consider implementing it as a static method in an object (we can still have a convenience implicit) so that its usable from Java.